app/models/account.rb

```diff
@@ -180,6 +180,10 @@ class Account < ApplicationRecord
 
   update_index('accounts', :self)
 
+  def actor_type_application?
```

Comment:
This is a nice readability improvement even if we dont want/need to move to an `enum` here -- nice!